src/utils/screen_utils.py

```diff
@@ -144,10 +144,11 @@ def read_device_state(path_to_device_state: str) -> Dict[str, Any]:
     except json.JSONDecodeError:
         logger.error('JSON Decode Error occurred reading from device state')
```

Comment:
Message: "JSON Decode Error occurred...", yeah. But what does that mean? Maybe a better error message would be `Unable to parse device state: Incorrect JSON formatting`.

Reply:
Totally, make it less cryptic. Changes made.